### DISCUSSION TITLE
Using three's post processing effect composer...

### DISCUSSION THREAD

**Discussion by gonzalle** [mkkellogg/GaussianSplats3D#139] [Ideas]
I noticed that playcanvas' editor rendering is slightly crisper (this could be good or bad, depending of the model).
In order to achieve the same effect, I just tried to add Three's effectComposer renderPass... without any success !... 
My question is : do you think that adding post effects is even possible on top of the splats ?
In any case, I think it should be great feature for color corrections, fx, etc...

**Comment by gonzalle** [comment]
Btw, my try was simply to add the effect composer like this :
After creating [my composer](https://threejs.org/docs/#manual/en/introduction/How-to-use-post-processing) and disabling viewer's `selfDrivenMode` I just added 
`        function update() {
            requestAnimationFrame(update);
            viewer.update();
            viewer.render();
            composer.render();
        }`
  what gave me a bunch of errors....

**Comment by mkkellogg** [comment]
Sorry for the late response, I haven't tried to use the effect composer with my viewer so I'm not sure if it's something that can be done, or what the right way to do it is. When I have a spare moment, I will take a stab at it :)
